message_app_client/src/App.js

```diff
@@ -1,25 +1,19 @@
-import logo from './logo.svg';
 import './App.css';
+import ConversationDetailPanel from './components/ConversationDetailPanel';
+import ConversationContainer from './components/conversation';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <ConversationContainer conversations={[]}/>
```

Comment:
The top component should be `ConversationPanel`. `ConversationContainer` is a child of `ConversationPanel`